backend/oasst_backend/utils/hugging_face.py

```diff
@@ -46,4 +46,6 @@ async def post(self, input: str) -> Any:
         # Get the response from the API call
         inference = await response.json()
 
+        await session.close()
```

Comment:
Could you please use a `async with` statement for the session (it will then be closed automatically) so that the client is also closed in case of an exception?

Reply:
Sure thanks! Done